faqs/galaxy/datatype_tabular.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Indentifying and Formatting Tabular Dataset 
+area: Datatypes
+description: Format help for Tabular/BED/Interval Datasets
+layout: faq          
+---
+ 
+
+- A Tabular datatype is human readable and has tabs separating data columns.
+  - Note: tabular data is different from comma seperated data (.csv)
+1. Common tabular datatypes are `.bed`, `.gtf`, `.interval`, or `.txt`.
+2. The datatype metadata attribute can be directly reassigned to tabular format data.
+3. Click the pencil icon {% icon galaxy-pencil %} to reach the **_Edit Attributes_** form. 
+   - Change the datatype (3rd tab) and save
+   - Label columns (1st tab) and save
+   - Metadata will be assigned, then the dataset can be used.
```

Comment:
these 3 could maybe be converted into steps

Reply:
Done.